src/index.js

```diff
@@ -3,29 +3,25 @@ import axios from "axios";
 import fs from "fs";
 import shell from "shelljs";
 
+function commits_for(contribution) {
+  return `GIT_AUTHOR_DATE=${contribution.date}T12:00:00 GIT_COMMITER_DATE=${contribution.date}T12:00:00 git commit --allow-empty -m "Rewriting History!" > /dev/null\n`.repeat(contribution.count)
+}
+
 export default async (input) => {
   const res = await axios.get(
     `https://github.com/users/${input.username}/contributions?tab=overview&from=${input.year}-12-01&to=${input.year}-12-31`
   );
 
   // Gathers all the squares from GitHub contribution graph.
-  const allSquares = parse(res.data).querySelectorAll("[data-count]");
-
-  const script = allSquares
-    .reduce((fullCommand, contribution) => {
-      const count = contribution._rawAttrs["data-count"];
-      const date = contribution._rawAttrs["data-date"];
-
-      // Only care about html elements of squares with 1+ contributions for that day.
-      if (count > 0) {
-        return (fullCommand +=
-          `GIT_AUTHOR_DATE=${date}T12:00:00 GIT_COMMITER_DATE=${date}T12:00:00 git commit --allow-empty -m "Rewriting History!" > /dev/null\n`.repeat(
-            count
-          ));
-      }
-      return fullCommand;
-    }, "#!/bin/bash \n")
-    .concat("", "git pull origin main\ngit push -f origin main");
+  const script = parse(res.data).querySelectorAll("[data-count]")
+    .map(el => { return {date: el.attributes["data-date"], count: parseInt(el.attributes["data-count"])}} )
```

Comment:
The original code used `_rawattr`, and I think that was to directly fetch the integer value. Neither `_rawattr` nor `parseInt(el.attributes)` would work if github decide to put not integer values in `data-count` but that's unlikely.

Reply:
_rawAttra included all the metadata regarding of it being a number when I looked at its shape. Shouldn’t be a problem 😊

Reply:
In general, methods and properties that start with underscore are pseudo-private and shouldn't be relied on. But for this example, it's much of a muchness